test/JDBC/src/main/java/com/sqlsamples/CompareResults.java

```diff
@@ -138,7 +156,14 @@ static void processResults(Statement stmt, BufferedWriter bw, int resultsProcess
             if ((!resultSetExist) && (updateCount == -1)) {
                 break;
             }
-
+            if (warningExist) {
```

Comment:
Why do we need an additional flag `warningExist`? Can we just check whether `stmt.getWarnings()` is not null?